examples/ts/sol/utils/nonce-account-creation/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "sol-nonce-account-creation",
+  "version": "1.0.0",
+  "description": "Nonce account generator",
+  "scripts": {
+    "createAccount": "ts-node create-account.ts",
+    "generateNonceAccounts": "ts-node generate-nonce-accounts.ts"
+  },
+  "dependencies": {
+    "@bitgo/sdk-coin-sol": "2.15.0",
+    "@bitgo/statics": "24.0.0",
+    "@solana/web3.js": "1.29.2"
+  },
+  "devDependencies": {
+    "ts-node": "10.9.1",
+    "typescript": "5.1.6"
+  }
+}
```

Comment:
Dependencies pull from root. You should be able to run any example without this.